src/index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
+  </head>
+  <body>
+    Hello, world!
+  </body>
+</html>
```

Comment:
add newline at the end of the file
https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline